src/Features/x42.Features.xServer/Api/Controllers/xServerController.cs

```diff
@@ -208,6 +208,7 @@ public IActionResult CreatePriceLock([FromBody] CreatePriceLockRequest priceLock
             try
             {
                 var result = this.xServerManager.CreatePriceLock(priceLockRequest);
+                this.xServerManager.RelayPriceLock(result);
```

Comment:
This will be fine for now, but this will need to change long term (A todo?). This could be slow, as creating a price lock will be waiting to loop through all of T3 xservers to relay this.